docs/tables/aws_cloudtrail_event_data_store.md

```diff
@@ -0,0 +1,54 @@
+# Table: aws_cloudtrail_event_data_store
+
+AWS CloudTrail Event data store stores data for up to seven years, or 2557 days. By default, event data is retained for 2557 days, and termination protection is enabled for an event data store.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  event_data_store_arn,
+  status,
+  created_timestamp,
+  multi_region_enabled,
+  organization_enabled,
+  termination_protection_enabled
+from
+  aws_cloudtrail_event_data_store;
+```
+
+### List cloudtrail event stores which are not enabled
```

Comment:
```suggestion
### List event data stores which are not enabled
```

We don't need the service name (unless it's to disambiguate between resource names together), and we should include the full resource name